src/draw/nema_gfx/lv_draw_nema_gfx_img.c

```diff
@@ -0,0 +1,263 @@
+/* TSI 2023.gen */
+// -----------------------------------------------------------------------------
+// Copyright (c) 2008-23 Think Silicon Single Member PC
+// Think Silicon Single Member PC Confidential Proprietary
+// -----------------------------------------------------------------------------
+//     All Rights reserved - Unpublished -rights reserved under
+//         the Copyright laws of the European Union
+//
+//  This file includes the Confidential information of Think Silicon Single
+//  Member PC The receiver of this Confidential Information shall not disclose
+//  it to any third party and shall protect its confidentiality by using the
+//  same degree of care, but not less then a reasonable degree of care, as the
+//  receiver uses to protect receiver's own Confidential Information. The entire
+//  notice must be reproduced on all authorized copies and copies may only be
+//  made to the extent permitted by a licensing agreement from Think Silicon
+//  Single Member PC.
+//
+//  The software/data is provided 'as is', without warranty of any kind,
+//  expressed or implied, including but not limited to the warranties of
+//  merchantability, fitness for a particular purpose and noninfringement. In no
+//  event shall Think Silicon Single Member PC be liable for any claim, damages
+//  or other liability, whether in an action of contract, tort or otherwise,
+//  arising from, out of or in connection with the software.
+//
+//  For further information please contact:
+//
+//                    Think Silicon Single Member PC
+//                    http://www.think-silicon.com
+//                    Patras Science Park
+//                    Rion Achaias 26504
+//                    Greece
+// -----------------------------------------------------------------------------
+
+/**
+ * @file lv_draw_nema_gfx_fill.c
+ *
+ */
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "lv_draw_nema_gfx.h"
+
+#if LV_USE_NEMA_GFX
+
+
+/**********************
+ *  STATIC PROTOTYPES
+ **********************/
+
+static uint32_t lv_cf_to_nema(lv_color_format_t cf);
+
+static void _draw_nema_gfx_tile(lv_draw_unit_t * draw_unit, const lv_draw_image_dsc_t * dsc, const lv_area_t * coords);
+
+static void _draw_nema_gfx_img(lv_draw_unit_t * draw_unit, const lv_draw_image_dsc_t * dsc, const lv_area_t * coords);
+
+static uint32_t skip_pallete(lv_color_format_t cf);
+
+/**********************
+ *  STATIC FUNCTIONS
+ **********************/
+
+static uint32_t skip_pallete(lv_color_format_t cf)
```

Comment:
It's not used, let's remove it.

Reply:
It will be used in next update for Look Up Table (LUT) Textures, or indexed images as referred in LVGL

Reply:
Let's add them in next update. I suspect there is a warning.